coin-flip/index.html

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en-US">
 <head>
 <title>Coin Flipper | My Digital Spot</title>
 <link rel="stylesheet" href="index.css">
 <script src="script.js"></script>
 </head>
-<center>
-<h1 class="header">Coin Flipper</h1>
-<div id="coin">
-<h1>Flip that coin!</h1>
-</div>
-<div id="coinimg"></div>
-<button onclick="getRandNumber(2)">Flip it again!</button>
-<div class="footer"><p>A simple coin flip app with clean design and nice colors. By Isaac Fiedler 2021. See https://github.com/chillzeus</p></div>
-</center>
+<body>
+    <h1 class="header">Coin Flipper</h1>
+    <div id="coin">
+    <h1>Flip that coin!</h1>
+    </div>
+    <div id="coinimg"></div>
+    <button onclick="flipCoin()">Flip it again!</button>
```

Comment:
I changed the call to `flipCoin()` since this way it's clearer what you're doing.

Reply:
Yeah, I do want it to run onload it's just right now there isn't anything in the function since it was only a draft.